File: Ejercicios de Semana Santa/ejercicios de semana santa php/Three php/Library/SegundaLey.php

<?php
require_once "LeyNewton.php";

class SegundaLey extends LeyNewton {
    private $m;
    private $a;
    public function ingresarDatos() {
        echo "Ingrese la masa (kg): ";
        $this->m = floatval(trim(fgets(STDIN)));
        echo "Ingrese la aceleración (m/s²): ";
        $this->a = floatval(trim(fgets(STDIN)));
    }
    public function calcular() {
        $f = $this->m * $this->a;
        echo "Fuerza aplicada: " . number_format($f, 2, ",", ".") . " N\n";
    }
}
?>

File: Ejercicios de Semana Santa/ejercicios de semana santa java/Three java/Library/SegundaLey.java

package Library;

import java.util.Scanner;

public class SegundaLey extends LeyNewton {
    double m, a;
    public void ingresarDatos() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese la masa (kg): ");
        m = sc.nextDouble();
        System.out.print("Ingrese la aceleración (m/s²): ");
        a = sc.nextDouble();
    }
    public void calcular() {
        double f = m * a;
        System.out.printf("Fuerza aplicada: %.2f N\n", f);
    }
}
